conans/client/toolchain/msbuild.py

```diff
@@ -50,6 +58,8 @@ def __init__(self, conanfile):
         self._conanfile = conanfile
         self.preprocessor_definitions = {}
 
+    filename = "conan_toolchain.props"
```

Comment:
This has changed in develop already, doesn't conflict because the definition is above. I will solve the conflict after merge